scss/modules/_site-header.scss

```diff
@@ -1,7 +1,21 @@
 // Site Header
-// 
+//
 // Styleguide modules.site-header
 
+// Staging banner
+.banner {
+  background-color: $navy;
+  color: $cream;
+  font-family: $sans-serif;
+  padding: 1rem 0;
+
+  a {
+    color: $cream;
```

Comment:
Is the idea that scss-lint wants (some) properties in alphabetical order? I wouldn't miss this warning if we lost it.